CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [30.3.0]
+### Added
+- New functionality added to the CollectiblesController to verify the transaction state of a collectible ([#762](https://github.com/MetaMask/controllers/pull/762)).
+  - New method added to the CollectiblesController to search a Collectible by smart contract address and token ID (`findCollectibleByAddressAndTokenId`).
+  - New method added to the CollectiblesController to update the state of a Collectible (`updateCollectible`).
+  - New method added to the CollectiblesController to reset the transaction status of a Collectible (`resetCollectibleTransactionStatusByTransactionId`).
+- New method called `validatePassword` is added to KeyringController to validate the password ([#884](https://github.com/MetaMask/controllers/pull/884))
+
+### Changed
+- Exporting `formatIconUrlWithProxy` to format iconUrl Proxy for tokens from Swaps API ([#889](https://github.com/MetaMask/controllers/pull/889))
```

Comment:
Nit: Cleaned the grammar up to use imperative tense, for consistency. 

Also the `formatIconUrlWithProxy` seems like an addition to me, since we're exposing a new method.

```suggestion
- Add new functionality to the CollectiblesController to verify the transaction state of a collectible ([#762](https://github.com/MetaMask/controllers/pull/762)).
  - Add new method to the CollectiblesController to search a Collectible by smart contract address and token ID (`findCollectibleByAddressAndTokenId`).
  - Add new method to the CollectiblesController to update the state of a Collectible (`updateCollectible`).
  - Add new method to the CollectiblesController to reset the transaction status of a Collectible (`resetCollectibleTransactionStatusByTransactionId`).
- Add new method `validatePassword` to KeyringController to validate the password ([#884](https://github.com/MetaMask/controllers/pull/884))
- Add `formatIconUrlWithProxy` function for formatting iconUrl Proxy for tokens from Swaps API ([#889](https://github.com/MetaMask/controllers/pull/889))
```